examples/acmetool/hooks/01triggerRancherGen.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+#
+# Executed by acmetool when a certificate is updated
+# Ensures that /etc/nginx/certs/ is present, then
+# overwrite-copies all live SSL certs to that directory.
+# Triggers a single-run of rancher-gen to reload Nginx containers.
+
+case "$1" in
+ "live-updated" )
+    echo "[HOOK]: Processing live-updated hook"
+    mkdir -p /etc/nginx/certs
+    cp -RLf /var/lib/acme/live/* /etc/nginx/certs/
+    rancher-gen --config ${RGEN_CONFIG_ONETIME}
+    ;;
```

Comment:
This "new loop" is implemented already - 01triggerRancherGen.sh is deprecated and can be removed.